### DISCUSSION TITLE
An invalid response was received from the upstream server

### DISCUSSION THREAD

**Discussion by MinhDuc010194** [Kong/kong#7714] [Help] [answered]
I flow this link to create Kong ingress gateway `https://konghq.com/blog/kubernetes-ingress-api-gateway`
it works well until I got the last step `curl $PROXY_IP/foo` i got the message  "message": "An invalid response was received from the upstream server" instead of "msg": "Hello from the foo microservice".  what happens in this case? do I wrong or missing something?

**Comment by fffonion** [comment, marked as answer]
Consider it's a duplicate of #7716, will move the discussion in issue.
